File: backend-java/sahayaSetu/src/main/java/com/sahayaSetu/dtos/NgoResponseDto.java
package com.sahayaSetu.dtos;

import com.sahayaSetu.entities.enums.NgoStatus;
import lombok.Data;
import java.time.LocalDateTime;

@Data
public class NgoResponseDto {
    private Long ngoId;
    private String darpanId;
    private String name;
    private String email;
    private String contactPhone;
    private String addressLine1;
    private String city;
    private NgoStatus status;
    private LocalDateTime approvedAt;
    private LocalDateTime createdAt;
}

File: backend-dotnet/SahayaSetu/DTO/NgoResponseDto.cs
using SahayaSetu.Models;

namespace SahayaSetu.DTO
{
    public class NgoResponseDto
    {
        public int NgoId { get; set; }
        public string? Name { get; set; }
        public string? Email { get; set; }
        public string? Phone { get; set; }
        public string? Address { get; set; }
        public string? City { get; set; }
        public string? Status { get; set; }
        public DateTime? ApprovedAt { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}
